jobs/cronjob-truncate-cassandra-tag-metrics.yml

```diff
@@ -0,0 +1,168 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: cronjob-truncate-cassandra-tag-metrics
+  annotations:
+    description: Scheduled Task to Truncate Cassandra Tags Index
+    iconClass: icon-shadowman
+    tags: management,cronjob,truncate,cassandra
+objects:
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    annotations:
+      description: Keeps track of changes in the application image
+    name: ${NAME}
+    labels:
+      template: cronjob-truncate-cassandra-tag-metrics
+  spec:
+    lookupPolicy:
+      local: true
+- kind: BuildConfig
+  apiVersion: v1
+  metadata:
+    annotations:
+      description: Defines how to build the application
+    name: ${NAME}
+    labels:
+      template: cronjob-truncate-cassandra-tag-metrics
+  spec:
+    completionDeadlineSeconds: "1800"
+    output:
+      to:
+        kind: ImageStreamTag
+        name: ${NAME}:${TAG}
+    runPolicy: Serial
+    source:
+      git:
+        uri: "${SOURCE_REPOSITORY_URL}"
+        ref: "${SOURCE_REPOSITORY_REF}"
+      contextDir: "${CONTEXT_DIR}"
+    strategy:
+      dockerStrategy:
+        from:
+          kind: DockerImage
+          name: "${BASE_IMAGE}"
+      type: Docker
+    triggers:
+    - type: ConfigChange
+- kind: CronJob
+  apiVersion: batch/v1beta1
+  metadata:
+    name: "${JOB_NAME}"
+    labels:
+      template: cronjob-truncate-cassandra-tag-metrics
+  spec:
+    schedule: "${SCHEDULE}"
+    concurrencyPolicy: Forbid
+    successfulJobsHistoryLimit: "${{SUCCESS_JOBS_HISTORY_LIMIT}}"
+    failedJobsHistoryLimit: "${{FAILED_JOBS_HISTORY_LIMIT}}"
+    jobTemplate:
+      spec:
+        template:
+          spec:
+            containers:
+            - name: "${JOB_NAME}"
+              image: "${NAME}:${TAG}"
+              command: 
+              - "/bin/bash"
+              - "-c"
+              - "/usr/local/bin/truncate-cassandra-tag-metrics.sh"
+              env:
+              - name: METRICS_NAMESPACE
+                value: "${METRICS_NAMESPACE}"
+              - name: CASSANDRA_CONTAINER_PREFIX
+                value: "${CASSANDRA_CONTAINER_PREFIX}"
+            restartPolicy: Never
+            terminationGracePeriodSeconds: 30
+            activeDeadlineSeconds: 500
+            dnsPolicy: ClusterFirst
+            serviceAccountName: "${JOB_SERVICE_ACCOUNT}"
+            serviceAccount: "${JOB_SERVICE_ACCOUNT}"
+- kind: ClusterRoleBinding
+  apiVersion: v1
+  metadata:
+    name: system:cassandra-cleaner
+    labels:
+      template: "cronjob-truncate-cassandra-tag-metrics"
+  roleRef:
+    name: cluster-admin
```

Comment:
Do we want to create a dedicated cluster role scoped for this purpose instead of a cluster-admin role?